lib/buildkite/builder/commands/run.rb

```diff
@@ -15,8 +15,13 @@ def run
 
           # This entrypoint is for running on CI. It expects certain environment
           # variables to be set. It also uploads the pipeline to Buildkite.
-          log.info "#{'+++ ' if Buildkite.env}🧰 " + 'Buildkite Builder'.color(:springgreen) + " ─ #{relative_pipeline_path.to_s.yellow}"
-          Pipeline.new(pipeline_path, logger: log).upload
+          log.info "+++ 🧰 #{'Buildkite Builder'.color(:springgreen)} ─ #{relative_pipeline_path.to_s.yellow}"
+
+          if Buildkite::Pipelines::Command.meta_data(:exists, Builder::META_DATA.fetch(:job))
```

Comment:
Let's make sure commands are globally stubbed in CI. Maybe a support or a configuration flag on `Command`.